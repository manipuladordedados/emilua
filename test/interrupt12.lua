-- Read invalid interruption_caught on some detached fiber
fib = spawn(function()
    this_fiber.yield()
end)
this_fiber.yield()
fib:detach()
print(fib.interruption_caught)
