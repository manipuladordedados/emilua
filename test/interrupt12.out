Main fiber from VM 0x1 panicked: 'Invalid argument'
stack traceback:
	input.lua:7: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
